sources/niLang/src/_build.ham

```diff
@@ -25,6 +25,7 @@ if $(FEATURES) >= 15 {
   ### PCRE ###
   SRC_EXTRA += [ tkPkgSrcSubDir pcre ] ;
 }
+ObjectDefines $(SRC_EXTRA) : MBEDTLS_ECP_RESTARTABLE ;
```

Comment:
Can't we move that to a config header in mbedtls? This seems like the wrong place for this, ideally the code should be without any external defines. The exception is only for extreme cases where its impossible or legacy build files that are that way...